### PR TITLE
Some small changes to the Dockerfile.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -124,6 +124,7 @@ ENV PATH="/usr/local/miniconda/bin:$PATH" \
     LC_ALL="C.UTF-8" \
     PYTHONNOUSERSITE=1
 
+ENV FOO=1
```
Comment (arokem) [inline Dockerfile]: This one in particular mystifies me. But if it's not there, I get a failure to import matplotlib a couple of steps down (in the font caching step).

File: Dockerfile [hunk 2]
```diff
@@ -191,3 +193,5 @@ LABEL org.label-schema.build-date=$BUILD_DATE \
       org.label-schema.vcs-url="https://github.com/nipreps/dmriprep" \
       org.label-schema.version=$VERSION \
       org.label-schema.schema-version="1.0"
+
+COPY license.txt /usr/local/license.txt
```
Comment (oesteban) [inline Dockerfile]: what license is this?

Comment (arokem) [reply Dockerfile]: It's the Freesurfer license. Is there a better way to get that in there?

Comment (oesteban) [reply Dockerfile]: Yup, there are a couple of options:
1. Use the ``--fs-license`` argument + mounting the file somewhere
2. Use the ``-e FS_LICENSE=/path/in/container/license.txt`` + mounting the file

Comment (arokem) [reply Dockerfile]: OK - I have removed this line.

File: Dockerfile [hunk 3]
```diff
@@ -169,13 +170,12 @@ ARG VERSION
...
 RUN echo "${VERSION}" > /src/dmriprep/dmriprep/VERSION && \
     echo "include dmriprep/VERSION" >> /src/dmriprep/MANIFEST.in && \
-    pip install --no-cache-dir "/src/dmriprep[all]"
+    cd /src/dmriprep && \
+    pip install --no-cache-dir .[all]
 
 RUN find $HOME -type d -exec chmod go=u {} + && \
     find $HOME -type f -exec chmod go=u {} +
 
-ENV IS_DOCKER_8395080871=1
```
Comment (arokem) [inline Dockerfile]: This is also related to #51

File: Dockerfile (hunk 1 above)
Comment (oesteban) [reply Dockerfile]: What is the exact error?

Comment (oesteban) [reply Dockerfile]: (I'm going to merge because this is not really relevant)

Comment (arokem) [reply Dockerfile]: I get an import error when importing matplotlib on https://github.com/nipreps/dmriprep/blob/master/Dockerfile#L154, as though the installation on https://github.com/nipreps/dmriprep/blob/master/Dockerfile#L129 didn't work.

**Pull request merged**